clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStats.java

```diff
@@ -168,6 +170,7 @@ public IngestionStats(VeniceServerConfig serverConfig) {
         localMetricRepository,
         METRIC_CONFIG,
         NEARLINE_LOCAL_BROKER_TO_READY_TO_SERVE_LATENCY);
+    transformerLatencySensor = new WritePathLatencySensor(localMetricRepository, METRIC_CONFIG, TRANSFORMER_LATENCY);
```

Comment:
We need to register this sensor.  But ideally, we should only register this sensor if the transformer is passed and is being updated.

Let's wrap the register in a function and then invoke that function in the ingestion task if a transformer is passed.

Reply:
@ZacAttack Do you mind re-running the tests, I think the integration test A is being flaky

Reply:
And also let me know if this looks good to you